scripts/css-alias.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Script creates alias for CSS specs
+ * see https://github.com/tobie/specref/issues/563
+ */
+const { promises: fs } = require("fs");
+const w3cSpecsPath = require("path").resolve(__dirname, "../refs/w3c.json");
```

Comment:
var current = helper.readBiblio(FILE);
```suggestion
const helper = require('./helper');
const FILENAME = "w3c.json";
const obj = helper.readBiblio(FILENAME);
```